sql/information_schema/information_schema.go

```diff
@@ -1421,7 +1421,14 @@ func statisticsRowIter(ctx *Context, c Catalog) (RowIter, error) {
 							collation = "A"
 
 							// TODO : cardinality is an estimate of the number of unique values in the index.
-
+//							cardinality = 10000
+
+							if st, ok := index.(StatisticsIndex); ok {
+                               		                cardinality = st.RowCount()
+                      		                       if err != nil {
+                       		                               return nil, err
+			                                             }	
```

Comment:
This doesn't seem to be checking any errors for the code added in this PR. `err` hasn't been written to since line 1373.